kano_updater/scenarios.py

```diff
@@ -361,6 +361,7 @@ def beta_210_to_beta_220(self):
 
     def beta_220_to_beta_230(self):
         install('rsync')
+        run_cmd_log('kano-apps install --no-gui powerup')
```

Comment:
We've done in this way before https://github.com/KanoComputing/kano-updater/blob/jessie/kano_updater/scenarios.py#L287-L288 and https://github.com/KanoComputing/kano-updater/blob/jessie/kano_updater/scenarios.py#L327

I could change it if you want, just to be sure